netflix/src/components/details/DetailTop.tsx

```diff
@@ -0,0 +1,24 @@
+import Image from 'next/image';
+
+interface DetailTopProps {
+  poster_path: string | null;
+}
+
+export default function DetailTop({ poster_path }: DetailTopProps) {
+  return (
+    <section className="w-full h-[415px] relative">
+      <div className="absolute inset-0">
+        {poster_path && (
+          <Image
+            fill
+            src={`https://image.tmdb.org/t/p/original${poster_path}`}
+            alt="title"
+            style={{ objectFit: 'cover' }}
+            priority
+          />
+        )}
+      </div>
+      <div className="w-full h-full absolute bg-gradient-to-b from-transparent via-transparent to-black"></div>
```

Comment:
요 부분 컴포넌트로 생성해 주면 좀 더 코드가 깔끔해질 것 같아요 ㅎㅎ
Today.tsx 에서도 동일한 코드를 사용중이여서 재사용성도 좋아질 것 같습니다~~

```suggestion
   <GradientStyle />
```